.github/CODEOWNERS

```diff
@@ -12,41 +12,43 @@
 
 # Documentation
-/docs/ @grafana/docs-grafana
-/contribute/ @grafana/docs-grafana
-/docs/sources/developers/plugins/ @grafana/docs-grafana @grafana/plugins-platform-frontend @grafana/plugins-platform-backend
-/docs/sources/developers/plugins/backend/ @grafana/docs-grafana @grafana/plugins-platform-backend
 /.changelog-archive @grafana/docs-grafana
-CHANGELOG.md @grafana/docs-grafana
-CODE_OF_CONDUCT.md @grafana/docs-grafana
-CONTRIBUTING.md @grafana/docs-grafana
-GOVERNANCE.md @RichiH
-HALL_OF_FAME.md @grafana/docs-grafana
-ISSUE_TRIAGE.md @grafana/grafana-community-support
-LICENSE @torkelo
-LICENSING.md @torkelo
-MAINTAINERS.md @RichiH
-NOTICE.md @torkelo
-README.md @grafana/docs-grafana
-ROADMAP.md @torkelo
-SECURITY.md @grafana/security-team
-SUPPORT.md @torkelo
-UPGRADING_DEPENDENCIES.md @grafana/docs-grafana
-WORKFLOW.md @torkelo
-
-
-# Set up, dashboards/visualization, best practices: Chris Moyer
-# Alerting: Brenda Muir
-/docs/sources/administration/ @Eve832 @GrafanaWriter
-/docs/sources/alerting/ @brendamuir
-/docs/sources/dashboards/ @chri2547
-/docs/sources/datasources/ @Eve832 @GrafanaWriter
-/docs/sources/explore/ @Eve832 @GrafanaWriter
-/docs/sources/getting-started/ @chri2547
-/docs/sources/old-alerting/ @brendamuir
-/docs/sources/release-notes/ @Eve832 @GrafanaWriter
-/docs/sources/setup-grafana/ @chri2547
-/docs/sources/whatsnew/ @Eve832 @GrafanaWriter
+/CHANGELOG.md @grafana/docs-grafana
+/CODE_OF_CONDUCT.md @grafana/docs-grafana
+/CONTRIBUTING.md @grafana/docs-grafana
+/GOVERNANCE.md @RichiH
+/HALL_OF_FAME.md @grafana/docs-grafana
+/ISSUE_TRIAGE.md @grafana/grafana-community-support
+/LICENSE @torkelo
+/LICENSING.md @torkelo
+/MAINTAINERS.md @RichiH
+/NOTICE.md @torkelo
+/README.md @grafana/docs-grafana
+/ROADMAP.md @torkelo
+/SECURITY.md @grafana/security-team
+/SUPPORT.md @torkelo
+/UPGRADING_DEPENDENCIES.md @grafana/docs-grafana
+/WORKFLOW.md @torkelo
+/contribute/ @grafana/docs-grafana
+/devenv/README.md @grafana/docs-grafana
+
+# Technical documentation
+/docs/                                    @Eve832 @jdbaldry
+/docs/sources/                            @Eve832
+/docs/sources/administration/             @Eve832 @GrafanaWriter
+/docs/sources/alerting/                   @brendamuir
+/docs/sources/datasources/                @Eve832 @GrafanaWriter
+/docs/sources/explore/                    @Eve832 @GrafanaWriter
+/docs/sources/fundamentals                @chri2547
+/docs/sources/getting-started/            @chri2547
+/docs/sources/introduction/               @chri2547
+/docs/sources/old-alerting/               @brendamuir
+/docs/sources/release-notes/              @Eve832 @GrafanaWriter
+/docs/sources/setup-grafana/              @chri2547
+/docs/sources/upgrade-guide/              @chri2547
+/docs/sources/whatsnew/                   @chri2547
+/docs/sources/developers/plugins/         @Eve832 @grafana/plugins-platform-frontend @grafana/plugins-platform-backend
```

Comment:
My only thought here is that Joe is responsible for the plugin dev doc in the grafana repo. Maybe check with Fiona/Eve if he should be listed here.

Reply:
This is a tough one because not all of the plugins content is dev focused. It will be shard by Joe and Lyrissa. Can we tag me and Joe for now?

Reply:
Like this?

```suggestion
/docs/sources/developers/plugins/         @Eve832 @josmperez @grafana/plugins-platform-frontend @grafana/plugins-platform-backend
```